src/blocks/card.css

```diff
@@ -0,0 +1,102 @@
+.card {
+  width: 166px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  font-family: "Roboto", sans-serif;
+}
+
+.card__product-img {
+  display: block;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__product-name {
+  display: block;
+  margin-bottom: 4px;
+  font-family: inherit;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--base-text-color);
+}
+
+.card__product-code {
+  display: block;
+  margin-bottom: 16px;
+  font-family: inherit;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: var(--second-text-color);
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.card__review-count {
+  font-family: inherit;
+  font-style: normal;
+  font-weight: 400;
```

Comment:
font-style: normal;
  font-weight: 400; is default values, so they're redundant

Reply:
not fixed